core/templates/pages/skill-editor-page/navbar/skill-editor-navbar.directive.ts

```diff
@@ -148,7 +148,13 @@ angular.module('oppia').directive('skillEditorNavbar', [
                 () => {
                   ctrl.skill = SkillEditorStateService.getSkill();
                   $rootScope.$applyAsync();
-                }));
+                }),
+              SkillUpdateService.onPrerequisiteSkillChange.subscribe(
+                () => {
+                  $scope.$apply();
```

Comment:
nit: can we use `$applyAsync()`? It avoids errors when a digest loop is already in progress.

Reply:
okay